src/main/webapp/i18n/de/metis.json

```diff
@@ -94,7 +95,13 @@
                 "allPublicMessages": "Alle öffentlichen Nachrichten",
                 "searchResults": "Suchergebnisse für {{ search }}",
                 "mobileDisclaimer": "Bitte erst eine Konversation auswählen.",
-                "mobileDisclaimerCallToAction": "Konversation wählen"
+                "mobileDisclaimerCallToAction": "Konversation wählen",
+                "savedPosts": "Gespeicherte Nachrichten",
+                "progress": "In Bearbeitung",
+                "completed": "Abgeschlossen",
+                "archived": "Archiviert",
+                "emptySavedPosts": "Hier ist noch nichts. Speichere Nachrichten damit sie hier auftauchen.",
+                "deleteNoticeSavedPosts": "Abgeschlossene und archivierte Lesezeichen werden nach 100 Tagen gelöscht."
```

Comment:
"Abgeschlossenen Lesezeichen" sounds a bit weird, how would a "abgeschlossenes Lesezeichen" look like? Can we find a better word? "Gelesene", "Abgehakte", "Erledigte" (I like this one)?

Reply:
I changed it so that it says "Erledigte ... Lesezeichen" now. I also tried to change it in the sidebar and header sections. 👍